apps/bare-expo/android/settings.gradle

```diff
@@ -29,8 +29,23 @@ def reactNativePackage = providers.exec {
 
 dependencyResolutionManagement {
   versionCatalogs {
-    reactAndroidLibs {
+    libs {
       from(files(new File(reactNativePackage, "../gradle/libs.versions.toml")))
+      if (hasProperty('android.buildToolsVersion')) {
+        version("buildTools", findProperty('android.buildToolsVersion'))
+      }
+      if (hasProperty('android.minSdkVersion')) {
+        version("minSdk", findProperty('android.minSdkVersion'))
+      }
+      if (hasProperty('android.compileSdkVersion')) {
+        version("compileSdk", findProperty('android.compileSdkVersion'))
+      }
+      if (hasProperty('android.targetSdkVersion')) {
+        version("targetSdk", findProperty('android.targetSdkVersion'))
+      }
+      if (hasProperty('android.kotlinVersion')) {
+        version("kotlin", findProperty('android.kotlinVersion'))
+      }
```

Comment:
Adds overrides if needed - later, we can create our own `libs.versions.toml` file and combine them together